### PR TITLE
MAB cruise-specific cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Profiler Mooring ADCP cal csvs
Surface Mooring METLGR cal csvs

References:
mooring configs in Pioneer 20 folder in Vault
METBK_Module_Heights.xlsx in Sensors Notebook/METBK in Vault (same as NES)
3408-00201 cal csv procedure - updates in progress

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Pioneer 20 AM Updates](https://trello.com/c/22LKGqi6/232-pioneer-20-am-updates)

File: calibration/ADCPTG/CGINS-ADCPTG-24543__20240401.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+24543,CC_scale_factor1,0.45,constant; CP13EAPM-00001
+24543,CC_scale_factor2,0.45,constant
+24543,CC_scale_factor3,0.45,constant
+24543,CC_scale_factor4,0.45,constant
+24543,CC_bin_size,400,in cm
+24543,CC_dist_first_bin,901,in cm
+24543,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (reedan88) [inline calibration/ADCPTG/CGINS-ADCPTG-24543__20240401.csv:8]: Looks good.

File: calibration/ADCPSL/CGINS-ADCPSL-21497__20240401.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+21497,CC_scale_factor1,0.45,constant; CP14SEPM-00001
+21497,CC_scale_factor2,0.45,constant
+21497,CC_scale_factor3,0.45,constant
+21497,CC_scale_factor4,0.45,constant
+21497,CC_bin_size,800,in cm
+21497,CC_dist_first_bin,1695,in cm
+21497,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (reedan88) [inline calibration/ADCPSL/CGINS-ADCPSL-21497__20240401.csv:8]: Looks good.

**Review by reedan88 — APPROVED** [review]
Looks good.

File: calibration/ADCPSL/CGINS-ADCPSL-18445__20240401.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+18445,CC_scale_factor1,0.45,constant; CP14NEPM-00001
+18445,CC_scale_factor2,0.45,constant
+18445,CC_scale_factor3,0.45,constant
+18445,CC_scale_factor4,0.45,constant
+18445,CC_bin_size,800,in cm
+18445,CC_dist_first_bin,1695,in cm
+18445,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-18445__20240401.csv:8]: looks good

File: calibration/ADCPSL/CGINS-ADCPSL-21497__20240401.csv (hunk 2 above)
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-21497__20240401.csv:8]: looks good

File: calibration/ADCPTG/CGINS-ADCPTG-24543__20240401.csv (hunk 1 above)
Comment (cdobs) [inline calibration/ADCPTG/CGINS-ADCPTG-24543__20240401.csv:8]: looks good

File: calibration/METBKA/CGINS-METLGR-00022__20240401.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR022,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP10CNSM-00001 METBK1
+LGR022,CC_height_of_air_humidity_measurement_m,4.255,
+LGR022,CC_height_of_air_temperature_measurement_m,4.255,
+LGR022,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
+LGR022,CC_jcool,1,Constant
+LGR022,CC_jwarm,1,Constant
+LGR022,CC_zinvpbl,600,Constant
+LGR022,CC_use_velpt,1,Constant
```
Comment (cdobs) [inline calibration/METBKA/CGINS-METLGR-00022__20240401.csv:9]: looks good

**Review by cdobs — APPROVED** [review]
looks good to me

**Pull request merged**